### PR TITLE
ADEN-5573 VUAP tests clean-up

### PR DESCRIPTION
**Pull request by nandy-andy**
Changes:
* removed two classes:
** one testing deprecated creative template,
** second having only tests for sizes - moved to the only VUAP class,
* renamed things to be more descriptive,
* separated time progress tests as they're trouble-makers ;-)

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L216](https://github.com/Wikia/selenium-tests/blob/07cf0197a8ad33286cf5df663035db4ef30ec9cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L216): Remove this unused private "isOverlayNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L216](https://github.com/Wikia/selenium-tests/blob/47a31a9d9923b1dff352ab5bf56fb77150ab4ce5/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L216): Remove this unused private "isOverlayNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java [hunk 1]
```diff
@@ -1,53 +1,256 @@
...
   private static final long MAX_AUTOPLAY_MOVIE_DURATION = 40L;
+  private static final String FANDOM_ARTICLE_WESTWORLD_LINK = "http://adeng.fandom.wikia.com/articles/whats-coming-westworld-finale";
+  private static final String TNG_AD_REDIRECT_URL = "http://project43.wikia.com/wiki/DevTemplates/VUAP/TNG";
+  private static final String TURN_ON_RESOLVED_STATE = "resolved_state=true";
 
   @Test(
-          dataProviderClass = AdsDataProvider.class,
-          dataProvider = "adsVuapDesktop",
-          groups = "AdsVideoClosesWhenFinishPlaysOasis"
+    groups = {"AdsVuapDefaultState", "AdsVuapDefaultStateAutoplayOasis"},
```
Comment (bulaszenko) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java]: Maybe we should have only one group

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AutoplayVuap.java#L216](https://github.com/Wikia/selenium-tests/blob/656f524ebd1d0101a4fa1422e6bff740232d14fd/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java#L216): Remove this unused private "isOverlayNoVisible" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**